doc/faq.rst

```diff
@@ -123,8 +126,9 @@ The package relies on core developers using their free time to
 fix bugs, maintain code and review contributions.
 Any algorithm that is added needs future attention by the developers,
 at which point the original author might long have lost interest.
-Also see `this thread on the mailing list
-<https://sourceforge.net/p/scikit-learn/mailman/scikit-learn-general/thread/CAAkaFLWcBG+gtsFQzpTLfZoCsHMDv9UG5WaqT0LwUApte0TVzg@mail.gmail.com/#msg33104380>`_.
+Also see :ref:`new_algorithms_inclusion_criteria` and
```

Comment:
I would do something like this:
```
See also :ref:`new_algorithms_inclusion_criteria`. For a great read about 
long-term maintenance issues in open-source software, look at 
`the Executive Summary of Road and Bridges
<https://www.fordfoundation.org/media/2976/roads-and-bridges-the-unseen-labor-behind-our-digital-infrastructure.pdf#page=8>`_
```

Road and Bridges is 143 pages long so at least linking to the Execute Summary page is some kind of improvement. Better suggestions welcome for a better way to link to the Executive Summary (at the moment just the 8th page) or for another shorter reference.